### PR TITLE
Removal of OLD mech code (POWER LOADERS ARE VEHICLE CODE!)

### PR DESCRIPTION
**Pull request by Jalleo**
Removed all mech related code from the codebase apart from what is used on the map. I will change their paths at somepoint when I go to condense all this crud down later on.

Why It's Good For The Game

This means that we have less relative pathing and either someone can code good mechs or port the slightly better version from TG with a more easy time.

:cl: Jalleo
del: Removed old mech code
/:cl:

### CODE REVIEW

**Comment by Rohesie** [comment]
Merge conflict.

File: code/game/objects/structures/crates_lockers/largecrate.dm [hunk 1]
```diff
@@ -74,7 +74,7 @@
 		for(var/i = 0, i < num, i++)
 			new /mob/living/simple_animal/chick(loc)
 	..()
-
+/*
```
Comment (DominikPanic) [inline code/game/objects/structures/crates_lockers/largecrate.dm]: No commented out code please

**Comment by Jalleo** [comment]
I will fix this PR later this week.

**Comment by Rohesie** [comment]
Travis check.

**Comment by Rohesie** [comment]
Tested locally, didn't seem to cause issues.
Merge conflicts, though.

**Pull request merged**